.github/workflows/landing-builds.yml

```diff
@@ -12,8 +12,8 @@ on:
       - 'utils/docker/Dockerfile.*'
       - 'utils/scripts/install-*.sh'
       - 'utils/scripts/helpers/*'
-      - utils/ci
-      - ci
+      - utils/ci/*
+      - ci/*
```

Comment:
Whoops.  There are some subdirectories in `ci` but I don't think they matter.

Reply:
Meaning this change is wrong?

Reply:
I don't know if `ci/*` matches `ci/a/b` but I think all the files we want this to trigger on are directly in the `ci` directory so I think this is correct, even if I don't fully grok the details.

Reply:
I think the original intention here was `**` so I'll use that

Reply:
It's supposed to match https://github.com/daos-stack/daos/blob/master/ci/gha_helper.py#L12-L21 the idea being that those files are used for the hash key and therefore landings builds which populate the hash will trigger on those same files but actually I think github caching of docker layers have been broken for a while so that probably doesn't matter.

Without that, it's supposed to trigger on build/CI changes which would affect what/how it runs and I think just including the whole of those two directories is easiest.